docs/changelog.md

```diff
@@ -21,6 +21,13 @@ permalink: /changelog/
 * **feature-findinpage**
    * Find in Page now emits facts
 
+* **concept-push**
+  * 🆕 Added a new component for supporting push notifications.
```

Comment:
This PR doesn't add `concept-push`, it modifies it. Or was it not mentioned in previous changelogs?